larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -15,6 +15,7 @@
    ==============================================================================*/
 
 #include <gtest/gtest.h>
+#include <stdint.h>
```

Comment:
For C headers `xxx.h` there's usually a C++ version `cxxx` (without the `.h`), see [this page](https://en.cppreference.com/w/cpp/header#C_compatibility_headers), which puts standard library stuff in the `std` namespace.
```suggestion
#include <cstdint>
```

Reply:
Done in a8d1c1d.